gm4_scuba_gear/data/scuba_gear/functions/equipped.mcfunction

```diff
@@ -0,0 +1,3 @@
+execute if entity @s[tag=!gm4_in_water,gamemode=!spectator,nbt={Inventory:[{Slot:103b,tag:{gm4_scuba_gear:{item:"helmet"}}},{Slot:102b,tag:{gm4_scuba_gear:{item:"tank"}}}]}] at @s if block ~ ~1 ~ water run function scuba_gear:breathe
+execute if entity @s[tag=!gm4_in_water,gamemode=!spectator,nbt={Inventory:[{Slot:103b,tag:{gm4_scuba_gear:{item:"helmet"}}},{Slot:102b,tag:{gm4_scuba_gear:{item:"tank"}}}]}] at @s if block ~ ~1 ~ #scuba_gear:water_loggable[waterlogged=true] run function scuba_gear:breathe
+effect give @s[scores={gm4_sg_swim=1..},gamemode=!spectator,nbt={Inventory:[{Slot:100b,tag:{gm4_scuba_gear:{item:"flippers"}}}]}] dolphins_grace 1 0 true
```

Comment:
To avoid the same problem that weighted armor has, you should give the effect for 2 seconds instead.

Reply:
okay, it's been done